src/patternfly/components/Title/styles.scss

```diff
@@ -0,0 +1,119 @@
+$pf-c-title--lg--FontSize: $pf-global--FontSize--lg !default;
+$pf-c-title--lg--FontWeight: $pf-global--FontWeight--semi-bold !default;
+
+// md
+$pf-c-title--md--MarginTop: $pf-global--spacer--lg !default;
+$pf-c-title--md--MarginBottom: $pf-global--spacer--md !default;
+$pf-c-title--md--LineHeight: $pf-global--LineHeight--md !default;
+$pf-c-title--md--FontSize: $pf-global--FontSize--md !default;
+$pf-c-title--md--FontWeight: $pf-global--FontWeight--semi-bold !default;
+
+// sm
+$pf-c-title--sm--MarginTop: $pf-global--spacer--lg !default;
+$pf-c-title--sm--MarginBottom: $pf-global--spacer--xs !default;
+$pf-c-title--sm--LineHeight: $pf-global--LineHeight--md !default;
+$pf-c-title--sm--FontSize: $pf-global--FontSize--md !default;
+$pf-c-title--sm--FontWeight: $pf-global--FontWeight--semi-bold !default;
+
+.pf-c-title {
+  &.pf-is {
+    &-xxxxl {
+      font-size: $pf-c-title--xxxxl--FontSize;
+      font-weight: $pf-c-title--xxxxl--FontWeight;
+      line-height: $pf-c-title--xxxxl--LineHeight;
+      &.pf-has-margin {
+        margin-top: $pf-c-title--xxxxl--MarginTop;
+        margin-bottom: $pf-c-title--xxxxl--MarginBottom;
+      }
+    }
+    &-xxxl {
+      font-size: $pf-c-title--xxxl--FontSize;
+      font-weight: $pf-c-title--xxxl--FontWeight;
+      line-height: $pf-c-title--xxxl--LineHeight;
+      &.pf-has-margin {
+        margin-top: $pf-c-title--xxxl--MarginTop;
+        margin-bottom: $pf-c-title--xxxl--MarginBottom;
+      }
+    }
+    &-xxl {
+      font-size: $pf-c-title--xxl--FontSize;
+      font-weight: $pf-c-title--xxl--FontWeight;
+      line-height: $pf-c-title--xxl--LineHeight;
+      &.pf-has-margin {
+        margin-top: $pf-c-title--xxl--MarginTop;
+        margin-bottom: $pf-c-title--xxl--MarginBottom;
+      }
+    }
+    &-xl {
+      font-size: $pf-c-title--xl--FontSize;
+      font-weight: $pf-c-title--xl--FontWeight;
+      line-height: $pf-c-title--xl--LineHeight;
+      &.pf-has-margin {
+        margin-top: $pf-c-title--xl--MarginTop;
+        margin-bottom: $pf-c-title--xl--MarginBottom;
+      }
+    }
+    &-lg {
+      font-size: $pf-c-title--lg--FontSize;
+      font-weight: $pf-c-title--lg--FontWeight;
+      line-height: $pf-c-title--lg--LineHeight;
+      &.pf-has-margin {
+        margin-top: $pf-c-title--lg--MarginTop;
+        margin-bottom: $pf-c-title--lg--MarginBottom;
+      }
+    }
+    &-md {
+
+      font-size: $pf-c-title--md--FontSize;
+      font-weight: $pf-c-title--md--FontWeight;
+      line-height: $pf-c-title--md--LineHeight;
+      &.pf-has-margin {
+        margin-top: $pf-c-title--md--MarginTop;
+        margin-bottom: $pf-c-title--md--MarginBottom;
+      }
+    }
+    &-sm {
+      font-size: $pf-c-title--sm--FontSize;
+      font-weight: $pf-c-title--sm--FontWeight;
+      line-height: $pf-c-title--sm--LineHeight;
+      &.pf-has-margin {
+        margin-top: $pf-c-title--sm--MarginTop;
+        margin-bottom: $pf-c-title--sm--MarginBottom;
+      }
+    }
+
```

Comment:
and this empty line. I'd also add an empty line before each selector.

@dgutride We need a formatter!